From b6c334e70a201503f4a4995e505a75ab92fc7a82 Mon Sep 17 00:00:00 2001
From: Abhay Krishna Arunachalam <arnchlm@amazon.com>
Date: Mon, 13 Sep 2021 20:03:18 -0700
Subject: [PATCH] Replace promptui mod to mitigate LGPL dependency

Troubleshoot has a dependency on the manifoldco/promptui module, which in turn has a
dependency on juju/ansiterm module, which is LGPLv3 licensed. There is currently a 
PR to remove this dependency (https://github.com/manifoldco/promptui/pull/181) that
that has not been merged, but the PR author has updated their own fork to remove this
dependency, so we are using that module instead of the one from manifoldco.

Signed-off-by: Abhay Krishna Arunachalam <arnchlm@amazon.com>
---
 go.mod | 1 +
 1 file changed, 1 insertion(+)

diff --git a/go.mod b/go.mod
index 9f98c8c..9046b26 100644
--- a/go.mod
+++ b/go.mod
@@ -76,4 +76,5 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.20.2
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.2
 	sigs.k8s.io/controller-runtime => github.com/kubernetes-sigs/controller-runtime v0.8.3
+	github.com/manifoldco/promptui => github.com/nguyer/promptui v0.8.1-0.20210517132806-70ccd4709797
 )
-- 
2.33.0

